src/gui/updateurldialog.cpp

```diff
@@ -0,0 +1,55 @@
+#include "updateurldialog.h"
+
+#include <QMessageBox>
+#include <QTimer>
+// this include allows comparing QAbstractButton* and QPushButton*
```

Comment:
Does commenting includes really help?

Reply:
Qt is full of forward declarations. `QMessageBox` declares those classes, but you can't instantiate them. I can remove the comment.